### ISSUE TITLE
OpenSCAD with X11

### ISSUE THREAD

**Issue by fancsali** [termux/termux-packages#10187]
**Package description**

As **OpenSCAD** is already part of the **science-packages** repository, I'd propose to include the X11-related code as well, so one can use it via **Xserver Xsdl** on their Chromebooks, Android tablets and similar devices.

**Links**
- https://github.com/termux/science-packages/tree/master/packages/openscad
- https://github.com/termux/science-packages/issues/33

**Comment by robertkirkman** [comment]
@JulianBnder yes and i did it just now do you want it? please help me test it!

<img width="2235" height="1088" alt="Image" src="https://github.com/user-attachments/assets/aae3776d-6350-4121-ae6a-c7244ba88db4" />

@anonimo82 you wanted this also, right? do you know any others who wanted this?

after 6 years, I have arrived, implemented all required dependencies (that did not previously exist in termux), and finally built it for Termux, and on my device it's opening!

**Comment by robertkirkman** [comment]
- https://github.com/termux/termux-packages/pull/30351
